server/index.js

```diff
@@ -5,13 +5,20 @@ const PORT = 3001;
 const getFns = require('./controllers/getRequests.js');
 const faker = require('faker');
 const bodyparser = require('body-parser');
+const moment = require('moment');
+var monthTracker;
+var yearTracker;
+var currentMonth;
+var currentDay;
+var currentDate;
+var currentYear = 19;
```

Comment:
Probably doesn't matter for this project, but in general I think you wouldn't want to hard-code dates / years like this so you don't have to go back and refactor later